controllers/RssController.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+
+namespace app\controllers;
+
+
+use app\models\Extension;
+use app\models\News;
+use app\models\Wiki;
+use DateTime;
+use yii\base\Exception;
+use yii\helpers\Url;
+use yii\web\Controller;
+use Zend\Feed\Writer\Feed;
+
+class RssController extends Controller
+{
+    public function actionAll()
+    {
+        $feed = new Feed();
+        // Title of the feed
+        $feed->setTitle('Live News for Yii Framework');
+        // Link to the feed's target, usually a homepage:
+        $feed->setLink('http://www.yiiframework.com/');
+        // Link to the feed itself, and the feed type:
+        $feed->setFeedLink('http://www.yiiframework.com/rss.xml', 'rss');
+
+        // Feed description; only required for RSS:
+        $feed->setDescription('News, fresh extensions and wiki articles about Yii framework.');
+
+        /** @var News[]|Extension[]|Wiki[] $data */
+        $data = array_merge(
+            $this->getNews(),
+            $this->getExtensions(),
+            $this->getWikiPages()
+        );
+
+        asort($data);
+
+        $latest = new DateTime('@0');
+        foreach ($data as $datum) {
+            if ($datum instanceof News) {
+                $entry = $this->createNewsEntry($feed, $datum);
+            } elseif ($datum instanceof Extension) {
+                $entry = $this->createExtensionEntry($feed, $datum);
+            } elseif ($datum instanceof Wiki) {
+                $entry = $this->createWikiEntry($feed, $datum);
+            } else {
+                throw new Exception('Unable to create entry for datum of type ' . get_class($datum));
+            }
+            $feed->addEntry($entry);
+            $latest = $entry->getDateModified() > $latest ? $entry->getDateModified() : $latest;
```

Comment:
I assume it does not, you can simply use the first entry in the array to set modified date.